docs/topic/idle-culler.md

```diff
@@ -40,6 +40,12 @@ the users will not be culled alongside their notebooks and will continue to exis
 services.cull.users = False
 ```
 
+If named servers are in use, they are not removed after being culled.
```

Comment:
I make one edit during the rebase here.

Previously it said `If named servers are in use, remove the server after stopping it.` which didn't make sense to describe the default value of False for `remove_named_servers`.